pkg/controller/services.go

```diff
@@ -235,6 +235,9 @@ func (cont *AciController) staticServiceObjs() apicapi.ApicSlice {
 	// Service bridge domain
 	bdName := cont.aciNameForKey("bd", cont.env.ServiceBd())
 	bd := apicapi.NewFvBD(cont.config.AciVrfTenant, bdName)
+	if apicapi.ApicVersion >= "6.0(3.77a)" {
```

Comment:
I think 6.0(3.77a) is a Q/A release in preparation for 6.0.4. Assuming we want to only support this for 6.0(4x) and above, the comparison should be with a version number like this: 6.0(4x) as done here: https://github.com/noironetworks/aci-containers/blob/master/pkg/controller/controller.go#L676

Unfortunately, we don't know what "x" will be yet, and we will only know once its released. So this version comparison is tricky.

Reply:
Hence it is safe to compare against the known version on which it is supported. It will continue
 to work for 6.04 when released.

Reply:
okay, so we can keep the way it is